flox/aggregations.py

```diff
@@ -64,7 +65,7 @@ def generic_aggregate(
             f"Expected engine to be one of ['flox', 'numpy', 'numba']. Received {engine} instead."
         )
 
-    group_idx = np.asarray(group_idx, like=array)
+    group_idx = asarray(group_idx, like=array)
```

Comment:
Is this needed?

Reply:
I thought so, but running it again apparently not. Will remove.